dist/pythonlibs/riotctrl_ctrl/factory.py

```diff
@@ -0,0 +1,58 @@
+# Copyright (C) 2021 Freie Universität Berlin
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+
+import os
+
+import riotctrl.ctrl
+
+from .native import NativeRIOTCtrl
+
+
+class RIOTCtrlFactory:
```

Comment:
This seems to be generic enough to go to riotctrl.

Reply:
I was thinking the same tbh. Also would be useful with https://github.com/RIOT-OS/riotctrl/pull/13 when there is a `iotlabcli` adaptor for that.

Reply:
See https://github.com/RIOT-OS/riotctrl/pull/18